modules/testsuite/pom.xml

```diff
@@ -259,7 +262,10 @@
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
-          <argLine>${surefire.jvm.args} ${surefire.jvm.management.args} ${surefire.default-mgmt-serurity.args} ${ipVerArgs}</argLine>
+          <systemPropertyVariables>
+            <server.jvm.jpms.args>${server.jvm.jpms.args}</server.jvm.jpms.args>
+          </systemPropertyVariables>
+          <argLine>${surefire.jvm.args} ${surefire.jvm.management.args} ${surefire.default-mgmt-serurity.args} ${ipVerArgs} ${client.jvm.jpms.args} -Dlauncher.skip.jpms.properties=true</argLine>
```

Comment:
What's the purpose the launcher.skip.jpms.properties=true here ?  Who will read this property ?

Reply:
Purpose of this is to make sure the valid jpms properties are used, because it’s possible that launcher adds some properties that won’t be present in a tested wildfly server.
It's used to disable properties from `wildfly-arquillian-container` specifically `wildfly-core launcher`.

Reply:
@jedla97  I didn't see this is read in jbossws-cxf.  Can you please point me where it's used ?

Reply:
https://github.com/wildfly/wildfly-core/blob/main/launcher/src/main/java/org/wildfly/core/launcher/AbstractCommandBuilder.java#L65

Reply:
Well, the default options from [AbstractCommandBuilder.java#L65](https://github.com/wildfly/wildfly-core/blob/main/launcher/src/main/java/org/wildfly/core/launcher/AbstractCommandBuilder.java#L65) would be added if `launcher.skip.jpms.properties` is unset, or set to false. This MR set it to true. So default values from AbstractCommandBuilder won't be added, which is ok, because default values from this TS directly would be used.